lib/trace-object.js

```diff
@@ -12,8 +12,10 @@ exports.tracerOptions = tracerOptions;
 
 exports.sendTraceObject = function sendTraceObject(parentCtl) {
   if (cluster.isWorker) {
-    debug('starting trace object on worker %d', cluster.worker.id);
```

Comment:
Unrelated code, but I assume you added because you hit a problem. I suspect that the problem is that you were starting the tracer if it was licensed, but that other parts of the supervisor code were expecting the tracer to not start unless `enableTracing` was true, creating a conflict. If that wasn't the problem, what was?